fdbclient/FDBTypes.h

```diff
@@ -37,18 +37,23 @@ typedef StringRef ValueRef;
 typedef int64_t Generation;
 
 enum {
-	tagLocalitySpecial = -1,
+	tagLocalitySpecial = -1, // tag with this locality means it is invalidTag (id=0), txsTag (id=1), or cacheTag (id=2)
 	tagLocalityLogRouter = -2,
-	tagLocalityRemoteLog = -3,
+	tagLocalityRemoteLog = -3, // Q: definition?
 	tagLocalityUpgraded = -4,
 	tagLocalitySatellite = -5,
-	tagLocalityLogRouterMapped = -6,
+	tagLocalityLogRouterMapped = -6, // Q: definition? I guess it is used by pseudo tags
```

Comment:
@jzhou77  Is this the pseudo tag for the new backup?

Reply:
This is the pseudo tag used by log routers to pop the real LogRouter tag (i.e., `-2`).